### ISSUE TITLE
Allow required fields without providing validator, allow multiple validators

### ISSUE THREAD

**Issue by pierreis** [masumsoft/express-cassandra#79]
In certain cases, it is useful to set fields as required without enforcing any particular validation.
Currently, the `BaseModel._get_validators` method requires the user to provide a valid validator function (or, at least, a non-`undefined` field), even when `required` option is set (which could be seen as a validator by itself).

Current solution I see would be to set a dummy validator function along with `required = true`.
Is this simple case not handled by the ORM for a reason, or is it any better solution to achieve this?

Also, current validator declarations doesn't seem to allow for multiple validators. This would be particularly useful to provide a way to set error messages dependent of the error itself instead of just the value. Also, this would add a lot of flexibility when using libraries such as [validator](https://www.npmjs.com/package/validator).

An example API could be the following:

```javascript
module.exports = {
  //... other properties hidden for clarity
  age: {
    type: "int",
    rule: {
      required: true,
      validators: [
        { 
          validator: function (value) { return value > 0; },
          message: function (value) { return 'Age must be greater than 0. You provided ' + value; }
        },
        {
          validator: function (value) { return value < 100; }, 
          message: 'You\'re not that old!'
        }
      ]
    }
  }
}
```

**Comment by masumsoft** [comment]
You are right that the validator function must be optional. The `required: true` parameter were introduced much later than the validator method system. You could possibly mimic the required validation in a validator function. So I guess later code supporting the required param introduced this bug. Btw, I've worked on a simple fix on it. 

But about multiple validators, can't we just use a single validator to achieve the same result?

```js
module.exports = {
  //... other properties hidden for clarity
  age: {
    type: "int",
    rule: {
      required: true,
      validator: { 
        validator: function (value) {
          return (value > 0 && value <100);
        },
        message: function (value) {
          if (value <=0 ) return 'Age must be greater than 0. You provided ' + value;
          else return 'You\'re not that old!';
        }
      },
    }
  }
}
```

**Comment by masumsoft** [comment]
optional validator function fix published in v1.6.5

**Comment by pierreis** [comment]
For multiple validators, yes, definitely that can actually be achieved as you propose.
However, I would tend to find it a bit counterintuitive, as this basically implies to run the tests twice: once in the validator function to validate the input, and one in the message function to actually find out where it failed.

In the end, I would tend to support a validation model much closer to the one in Mongoose – multiple validators, each with its message, like the API I proposed above (async validators would be even better, but they are hardly compatible with the set-time validation).

This modification could be done without introducing any API changes – current validator style would simply be appended to the validators array.

**Comment by masumsoft** [comment]
Yah your point is valid, multiple validators would be more elegant in this case. I'm re-opening this one and will implement multiple validator support.

**Comment by masumsoft** [comment]
58fb51948f80ac8ba0d79b34be8049fe2dbe52d9 fixes the issue, published in version 1.7.0
